VultisigApp/VultisigApp/Chains/THORChainSwaps.swift

```diff
@@ -146,6 +158,13 @@ class THORChainSwaps {
             return try ATOMHelper().getSignedTransaction(vaultHexPubKey: vaultHexPublicKey, vaultHexChainCode: vaultHexChainCode, inputData: inputData, signatures: signatures)
         case .ripple:
             return try RippleHelper.getSignedTransaction(keysignPayload: keysignPayload, signatures: signatures)
+        case .tron:
+            
+            guard let vault = vault else {
+                throw HelperError.runtimeError("not support")
+            }
+            
+            return try TronHelper.getSignedTransaction(keysignPayload: keysignPayload, signatures: signatures, vault: vault)
```

Comment:
please update TronHelper.getSignedTransaction to take in public keys ,and chain code instead